lib/taurus/cli/common.py

```diff
@@ -0,0 +1,149 @@
+#############################################################################
+##
+# This file is part of Taurus
+##
+# http://taurus-scada.org
+##
+# Copyright 2011 CELLS / ALBA Synchrotron, Bellaterra, Spain
+##
+# Taurus is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+##
+# Taurus is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+##
+# You should have received a copy of the GNU Lesser General Public License
+# along with Taurus.  If not, see <http://www.gnu.org/licenses/>.
+##
+#############################################################################
+
+"""
+This module provides the taurus Command Line Interface common options.
+
+It is based on the click module to provide commonly used flags/options.
+They are used by taurus commands and can be used by plugin to easily
+extend its functionality.
+
+Example 1: Add custom subcommand
+script::
+
+    import click
+
+    import taurus.cli.common
+
+
+    @click.command("bar")
+    @aurus.cli.common.poll_period
+    @aurus.cli.common.default_formatter
+    @aurus.cli.common.window_name("Super Bar")
+    def bar(poll_period, default_formatter, window_name):
+        ...
+
+
+    if __name__ == '__main__':
+        bar()
+
+Example 2: Add custom subcommands' group
+script::
+
+    import click
+
+    import taurus.cli.common
+
+
+    @click.group("foo")
+    def foo():
+        pass
+
+
+    @foo.command('cmd1')
+    @aurus.cli.common.models
```

Comment:
Typo in docstring: missing initial "t" in "@aurus.cli.common". Several occurrences.